src/content/extensions/multipleroutes/src/options.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+// Saves options.
+function saveOptions() {
+  var multiRoutes;
+  var nonProxiedUdp;
+  var radios = document.getElementsByName('routeselection');
+  for (var i = 0, length = radios.length; i < length; i++) {
+    if (radios[i].checked) {
+      multiRoutes = i < 1;
+      nonProxiedUdp = i < 2;
```

Comment:
Seems like a named constant or two would go a long way to readability:

var multiRouteIndex = 0;
var nonProxiedUdpIndex = 1;

This would help readability here, and then below when you restore the radio buttons.